sql/src/main/antlr/OpenSearchSQLLexer.g4

```diff
@@ -357,6 +358,8 @@ BIT_XOR_OP:                         '^';
 DOT:                                '.';
 LR_BRACKET:                         '(';
 RR_BRACKET:                         ')';
+LT_SQR_PRTHS:                       '[';
```

Comment:
where's this name coming from?
Just wondering why not something like: `LR_SQUARE_BRACKET` and `RR_SQUARE_BRACKET`